client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -130,16 +149,19 @@ export class PluginsBrowser extends Component {
 		const shortLists = {};
 		const fullLists = {};
 
-		visibleCategories.forEach( category => {
+		this.visibleCategories.forEach( category => {
 			shortLists[ category ] = PluginsListStore.getShortList( category );
 			fullLists[ category ] = PluginsListStore.getFullList( category );
 		} );
-
 		fullLists.search = PluginsListStore.getSearchList( search );
-		return {
-			shortLists: shortLists,
-			fullLists: fullLists,
-		};
+		const output = { shortLists, fullLists, installedPlugins: [] };
+
+		if ( this.props.selectedSite ) {
+			// getPlugins can return undefined
+			output.installedPlugins = PluginsStore.getPlugins( this.props.selectedSite ) || [];
```

Comment:
Could we avoid any mutations of objects? I think we could move that out:

```
const installedPlugins = selectedSite ? PluginsStore.getPlugins( selectedSite ) || [] : [];
const output = { shortLists, fullLists, installedPlugins };
```

(untested)

Reply:
We can now serve plugin recommendations for WPCOM simple sites, this commit actually became unnecessary. Reverting!